qemu/tests/sr_iov_hotplug.py

```diff
@@ -279,11 +336,10 @@ def _device_removed():
             for pci_num in xrange(pci_num_range):
                 msg = "start hot-deleting %sth pci device," % (pci_num + 1)
                 msg += " repeat %d" % (j + 1)
-                error.context(msg, logging.info)
+                logging.info(msg)
                 pci_del(-(pci_num + 1))
     finally:
         if params.get("enable_set_link", "yes") == "yes":
-            error.context("Re-enabling the primary link(s) of guest",
-                          logging.info)
+            logging.info("Re-enabling the primary link(s) of guest")
```

Comment:
just to remove autotest dependency, Now, I have replaced `error.contex`t to `error_context.context`